ui/stories/stubs/kapacitor.js

```diff
@@ -0,0 +1,16 @@
+const kapacitor = () => {
+  return ({
+    "id": "\"1\"",
+    "name": "kapa",
+    "url": "http://chronograf.influxcloud.net:9092",
+    "username": "hunter",
+    "password": "TMnN5JyM35H9m6Hc",
```

Comment:
Haha, wow... My bad. That was absolutely my password.
I'll get that changed right away. I'm not sure how to change it though?
I guess I wasn't expecting that to be kept plaintext in memory on the frontend? Is there any reason for that?